usaspending_api/settings.py

```diff
@@ -41,7 +41,8 @@
 SECRET_KEY = get_random_string()
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+# Defaults to False, unless DJANGO_DEBUG env var is set to a truthy value
+DEBUG = os.environ.get("DJANGO_DEBUG", "").lower() in ["true", "1", "yes"]
```

Comment:
@kwhickey need to verify there isn't a failsafe check in the deploy scripts which will fail or overwrite this

Reply:
@tony-sappe It does a regex match for anything after `=`, so we're good here (always want it to be `False` for deployed envs (non local envs)).

https://github.com/fedspendingtransparency/data-act-build-tools/blob/master/usaspending-deploy/usaspending-api-launch.yml#L211-L216